src/test/java/org/reaktivity/nukleus/http2/internal/bench/Http2ServerBM.java

```diff
@@ -94,7 +95,8 @@
             LangUtil.rethrowUnchecked(ex);
         }
 
-        reaktor = Reaktor.launch(configuration, n -> "http".equals(n), Http2Controller.class::isAssignableFrom);
+        //reaktor = Reaktor.launch(configuration, n -> "http".equals(n), Http2Controller.class::isAssignableFrom);
+        reaktor = null;
```

Comment:
This is a useful non-functional checkpoint on the initial implementation work.

Please get the benchmark into a running state before we merge this PR, you can see a working example in the `tcp` nukleus.

No need to spend effort on actually optimizing the code just yet, that can be covered in future PRs no problem.